mail_template_substitute/models/account_move.py

```diff
@@ -0,0 +1,35 @@
+# Copyright 2024 Sodexis
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+from odoo import models
+
+
+class AccountMove(models.Model):
+    _inherit = "account.move"
+
+    def action_invoice_sent(self):
```

Comment:
this should happen in https://github.com/OCA/OCB/blob/17.0/addons/account/models/account_move.py#L4047 already

Reply:
> this should happen in https://github.com/OCA/OCB/blob/17.0/addons/account/models/account_move.py#L4047 already

_get_mail_template() method will need to provide the External ID of the substitute template, but templates can be created via User Interface too, so external ID will not be available for those records (Do we need to create External Identifier record manually for those created via UI?). @hbrunn

Reply:
no, but don't override `action_invoice_sent`, override `action_send_and_print`

Reply:
okay, I understood @hbrunn